packages/iris-grid/src/IrisGrid.tsx

```diff
@@ -2915,6 +2915,9 @@ export class IrisGrid extends Component<IrisGridProps, IrisGridState> {
     if (copyOperation != null) {
       this.setState({ copyOperation: null });
     }
+    if (this.grid?.state.cursorRow != null) {
+      this.setState({ gotoRow: `${this.grid.state.cursorRow + 1}` });
+    }
```

Comment:
Good start, but if you click on a row in the grid, then press and hold down on the keyboard, instead of moving downward through the grid it will move focus to the "Go to value" field which is incorrect. We need to set the row without changing focus.

That looks to be caused by a `useEffect` in `GotoRow`. There isn't much detail in the original [Go to row spec](https://drive.google.com/drive/folders/1qEAoNMTI6lVknZfGy655_VAObb59M9--) or https://github.com/deephaven/web-client-ui/issues/616. Consider the usability of how/when we should focus, and if you are unsure/want some guidance, message in #ddl-team-web channel and we can discuss.